MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include molecule/provisioner/ansible *
```

Comment:
@Chedi doesn't pbr already include this automatically? What will it change?

Reply:
I think the relevant part of https://github.com/ansible/molecule/issues/1753 was:

> the ansible folder under provisioner is not included because it's not a python module.

`pbr` is doing everything else automatically AFAIK wrt. to the `MANIFEST.in`.

Reply:
Weird. Well, if everyone wants to merge it right now — go ahead. I personally vote for replacing `pbr` with the approach which is more native to Python ecosystem. It requires a bit of work for the migration but is not a blocker here. It can be done subsequently.